src/test/K6/src/tests/platform/storage/messageboxinstances.js

```diff
@@ -87,6 +94,22 @@ export default function(data) {
     });
     addErrorCount(success);
 
+    //Test to get instances based on filter parameters: search string (app title) from storage: SBL and validate the response
+    filters = {
+        "instanceOwner.partyId": partyId,
+        "searchString": "app",
+        "language": "nb"
+    };
+    res = sbl.searchSblInstances(runtimeToken, filters);
+    success = check(res, {
+        "Search instances by app title status is 200:": (r) => r.status === 200,
+        "Search instances app title matches:": (r) => {
+            var responseInstances = r.json();
+            return responseInstances.every(instance => instance.title.includes("app"));
```

Comment:
The test will probably always work based on the testdata, but remember that we try to match the app title in all languages.. So if you search for "name" you will get "endring av navn" instances returned if you are set to nb i the portal.

Reply:
yes, that is true. in this test case, it is coded to return instances in 'nb' and the test apps have 'app' in their names.
when a different app is to be searched or if someone accidentaly changes the name of the app in the text resources, this test will fail and has to modified